tests/samples/VarDefine.h

```diff
@@ -0,0 +1,11 @@
+extern int a;
+#define A a
+
+#define B b // ignored because there is no such variable
+
+extern int c;
+#define C c
+#undef C // removed
```

Comment:
When we will add support for `extern int c` (#70 this will still generate a `def c: native.CInt = extern`, right?

Reply:
Yes